tests/images/polar_alignment_utils_new.py

```diff
@@ -0,0 +1,60 @@
+from PIL import Image, ImageOps
+
+
+def fix_image_size(max_width, max_height, image):
+    """Resizes images while maintaining the same aspect ratio.
+
+    Args:
+        max_width: Image width
+        max_height: Image height
+        image: Image to be resized
+
+    Returns:
+        The resized image.
+    """
+    width_ratio = max_width / image.size[0]  # resized image ratios
+    height_ratio = max_height / image.size[1]
+
+    new_width = int(width_ratio * image.size[0])  # proportional width, height value calculations
+    new_height = int(height_ratio * image.size[1])
+
+    new_image = image.resize((new_width, new_height))
+    return new_image
+
+
+def grid_display(
```

Comment:
Generally want to try and name something after the action it is taking:

```suggestion
def overlay_grid_display(
```